wiring/src/spark_wiring_print.cpp

```diff
@@ -177,6 +182,11 @@ size_t Print::println(double num, int digits)
    return n;
  }
 
+size_t Print::println(const __FlashStringHelper* str)
+{
+  return println(reinterpret_cast<const char*>(str));
+}
+
```

Comment:
Can we add these to the `wiring/api` tests?  It might also be worthwhile to add tests for some of the other definitions to ensure they do what we expect such as:

```
    // Set SS to high so a connected chip will be "deselected" by default
    uint8_t port = digitalPinToPort(SS);
    uint8_t bit = digitalPinToBitMask(SS);
    volatile uint8_t *reg = portModeRegister(port);

    // if the SS pin is not already configured as an output
    // then set it high (to enable the internal pull-up resistor)
    if(!(*reg & bit)){
      digitalWrite(SS, HIGH);
    }
```

I'm sure there are lots of different ways these are used.  I'm hoping that they work as expected with Particle devices given a correct PIN definition.